src/java/org/apache/cassandra/tools/StandaloneJournalUtil.java

```diff
@@ -44,17 +50,12 @@
 import org.apache.cassandra.service.accord.serializers.Version;
 import org.apache.cassandra.tcm.ClusterMetadataService;
 
-import java.io.IOException;
-import java.nio.file.FileSystems;
-import java.nio.file.Files;
-import java.nio.file.Path;
-import java.nio.file.PathMatcher;
-import java.util.Collections;
-import java.util.HashMap;
-import java.util.HashSet;
-import java.util.Map;
-import java.util.Set;
-import java.util.concurrent.atomic.AtomicInteger;
+import accord.local.RedundantBefore;
+import accord.primitives.Timestamp;
+import accord.primitives.TxnId;
```

Comment:
nit: I think we've been putting `accord` imports at the top of the other `o.a.c.cassandra` imports.